Kitodo/src/test/java/org/kitodo/MockDatabase.java

```diff
@@ -873,12 +866,12 @@ private static void insertWorkpieceProperties() throws DAOException, DataExcepti
     }
 
     /**
-     * Clean database after class. Truncate all tables, reset id sequences and clear
-     * session.
+     * Clean database after class. Truncate all tables, reset id sequences and
+     * clear session.
      */
     public static void cleanDatabase() {
         Session session = Helper.getHibernateSession();
-        Transaction transaction = session.beginTransaction();
+        final Transaction transaction = session.beginTransaction();
```

Comment:
I'm not sure if this final here is necessary. Is there some reason to add it there?

Reply:
Checkstyle complains that the variable is declared to far away from its use and should be marked final then.

Reply:
So this warning from checkstyle can be ignored.

Reply:
I have not invented these rules, and I do not like all of them either. But since we have them for now we shoud stick to it.

Reply:
Like my comment above: check style rules should be reviewed and adjusted if needed.